### PR TITLE
Add table of variants and clt_version

### PR DESCRIPTION
**Pull request by arjunsalyan**
- [x] Add table of variants
- [x] Add table/chart of clt_verison

### CODE REVIEW

File: app/ports/templates/ports/port-detail/installation_stats.html [hunk 1]
```diff
@@ -256,10 +283,40 @@ <h4 class="modal-title">How are monthly installations calculated?</h4>
...
+
+            var data = google.visualization.arrayToDataTable([
+                ['OS Version', {% for version in clt_parent %}'{% if version.grouper == '' %}unknown{% else %}{{ version.grouper }}{% endif %}', {% endfor %} { role: 'annotation' } ],
+                {% for os in os_clt_parent %}
+                    ['OS: {{ os.grouper }}', {% for version in clt_parent %}{% for i in os.list %}{% if i.submission__clt_version == version.grouper %}{{ i.num }}{% endif %}{% endfor %}, {% endfor %} ''],
+                {% endfor %}
+        ]);
+
+        var options = {
+            'title': 'CLT Versions',
```
Comment (mojca) [inline app/ports/templates/ports/port-detail/installation_stats.html]: Let's spell that name out.

I would use singular for version all over the place, but that's unrelated

Comment (arjunsalyan) [reply app/ports/templates/ports/port-detail/installation_stats.html]: Okay, thanks.

**Comment by mojca** [comment]
Here's how the wrong sorting order of CLT (for the ld64 port) looks like:

image

**Comment by mojca** [comment]
I see that `xcode` uses `sortversion`:
```
{% regroup os_version_and_xcode_version|sortversion:"xcode_version" by xcode_version as xcode_parent %}
```
while clt uses `dictsort`:
```
{% regroup os_version_and_clt_version|dictsort:"clt_version" by clt_version as clt_parent %}
```

**Pull request merged**